pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -99,7 +103,10 @@ in (buildEnv {
     mkdir -p "$TEXMFCNF"
     if [ -e "$TEXMFDIST/web2c/texmfcnf.lua" ]; then
       sed \
+        -e "s,\(TEXMFOS[ ]*=[ ]*\)[^\,]*,\1\"$TEXMFROOT\",g" \
```

Comment:
What is the reason of the square brackets around the space? Do they do anything?

Reply:
I guess it used to be `[ \t]*`? I am not sure, really. Some code here is pretty old.